admin/controllers/school.js

```diff
@@ -363,7 +371,8 @@ const getSelectPupilNotTakingCheck = async (req, res, next) => {
     pupilsList: pupils,
     htmlSortDirection,
     arrowSortDirection,
-    highlight: []
+    highlight: [],
```

Comment:
If you are always passing down an empty array for the highlight, shouldn't you instead fallback in the corresponding ejs like this `highlight || []`

Reply:
@GeorgeChatzigiannis I didn't look into that code, all I did was adding a comma! Let's please move any refactoring / improvements to a separate PR.